website/www/site/content/en/contribute/_index.md

```diff
@@ -73,20 +73,33 @@ detail.
 
 ### Prerequisites
 
- - a GitHub account
- - a Linux, macOS, or Microsoft Windows development environment with Java JDK 8 installed
- - [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
-   changes locally
- - [Go](https://golang.org) 1.12 or later installed for Go SDK development
+ - A GitHub account.
+ - A Linux, macOS, or Microsoft Windows development environment
+      - Required *only* For Windows:
+         - Java JDK 8 installed
+         - [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
+   changes locally.
+ - For SDK Development:
+      - [Go](https://golang.org) 1.12 or later installed for Go SDK development
  - Python 3.6, 3.7, and 3.8. Yes, you need all three versions installed.
-    - pip, setuptools, virtualenv, and tox installed for Python development
- - for large contributions, a signed [Individual Contributor License
+      - Required *only* For Windows:
+        - pip, setuptools, virtualenv, and tox installed for Python development
+ - For large contributions, a signed [Individual Contributor License
    Agreement](https://www.apache.org/licenses/icla.pdf) (ICLA) to the Apache
    Software Foundation (ASF).
 
 
+### Configuration options
+You have two options for configuring your development environment:
+- Local: Debian-based Distribution
+    - Manual steps
+    - Automated script (*only* for Linux and macOS)
+- Container: Docker-based
 
+#### Local: Debian-based Distribution
+##### Manual steps
```

Comment:
I'm not sure that we should list manual steps here. I'd like some input from others in the community on this though to confirm. /cc @TheNeuralBit @griscz @ajamato 

Since we're listing the prerequisites above, users can install them based on whatever OS tools they want. We also link to the installation script, which a user could either use or reference.

Reply:
I'll try to figure out how to include it in the wiki as the following comments suggest.